src/main/java/Frontend/CommandLineInterface.java

```diff
@@ -1,7 +1,38 @@
 package Frontend;
 
+import java.util.ArrayList;
+import java.util.Arrays;
+
 public class CommandLineInterface {
+    private static final String[] acceptedCommands = {"-eq", "-save", "-domain", "-2d", "-3d",
+            "-load", "-graph"};
+
     public static void main(String[] args) {
-        System.out.println("Hello World!");
+        ArrayList<String> userInputs = new ArrayList<>(Arrays.asList(args));
+        if (userInputs.size() % 2 != 0) {
+            // There is an issue -- the (command, response) pair doesn't match up
+            // Add null at the end for now, and find the issue
+            userInputs.add(null);
```

Comment:
To avoid the repeated code, can stop the if-statement here and proceed with loop, no need the else+loop